CMakeLists.txt

```diff
@@ -113,7 +113,6 @@ target_link_libraries(${CMAKE_PROJECT_NAME} PUBLIC ${PLATFORM_LIBS})
 
 # Our ABI is not yet stable
 set_target_properties(${CMAKE_PROJECT_NAME} PROPERTIES SOVERSION 0unstable)
-target_compile_options(${CMAKE_PROJECT_NAME} PRIVATE "-fvisibility=hidden")
```

Comment:
Are we adding this elsewhere for linux/darwin?

Reply:
Yes, it's in AwsCFlags.cmake